docs/general/ddl/create-table.rst

```diff
@@ -1,47 +1,73 @@
-.. _sql_ddl_create:
+.. _ddl-create-table:
 
 ===============
 Creating tables
 ===============
 
+Tables are the basic building blocks of a relational database. A table can hold
+multiple rows (i.e., records), with each row having multiple columns and each
+column holding a single data element (i.e., value). You can :ref:`query <dql>`
+tables to :ref:`insert data <inserting_data>`, :ref:`select <sql_dql_queries>`
+(i.e., retrieve) data, and :ref:`delete data <dml_deleting_data>`.
+
 .. rubric:: Table of contents
 
 .. contents::
    :local:
 
-Basics
-======
 
-To create a table use the :ref:`sql-create-table` command. You must at least
-specify a name for the table and names and types of the columns.
+.. _ddl-create-table-definition:
+
+Table definition
+================
 
-See :ref:`data-types` for information about the supported data types.
+To create a table, use the :ref:`sql-create-table` :ref:`statement
+<gloss-statement>`.
 
-This query creates a simple table with two columns of type ``integer`` and
-``text``::
+At a minimum, you must specify a table name and one or more column
+definitions. A column definition must specify a column name and a corresponding
+:ref:`data type <data-types>`.
 
-    cr> create table my_table (
+Here's an example statement::
+
+    cr> CREATE TABLE my_table (
     ...   first_column integer,
     ...   second_column text
     ... );
     CREATE OK, 1 row affected (... sec)
 
-A table can be removed by using the :ref:`drop-table` command::
+This statement creates a table named ``my_table`` with two columns named
+``first_column`` and ``second_column`` with types :ref:`integer
+<data-type-numeric>` and :ref:`text <data-type-text>`.
+
+A table can be dropped (i.e., deleted) by using the :ref:`drop-table`
+statement::
 
-    cr> drop table my_table;
+    cr> DROP TABLE my_table;
     DROP OK, 1 row affected (... sec)
 
-The :ref:`drop-table` command takes the optional clause ``IF EXISTS`` which
-prevents the generation of an error if the specified table does not exist::
+If the ``my_table`` table did not exist, the ``DROP TABLE`` statement above
+would return an error message. If you specify the ``IF EXISTS`` clause, the
+instruction is conditional on the table's existence and would not return an
+error message::
 
-    cr> drop table if exists my_table;
+    cr> DROP TABLE IF EXISTS my_table;
     DROP OK, 0 rows affected (... sec)
 
+.. TIP::
+
+    By default, CrateDB will enforce the column definitions you specified with
+    the ``CREATE TABLE`` statement (what's known as a *strict* :ref:`column
+    policy <column_policy>`). However, you can configure the
+    :ref:`column_policy table parameter <column_policy>` to allow the dynamic
+    creation of new columns at query-time (what's known as a *dynamic* column
```

Comment:
changed to:

> By default, CrateDB will enforce the column definitions you specified with the CREATE TABLE statement (what’s known as a strict column policy).
>
> However, you can configure the column_policy table parameter so that the INSERT, UPDATE, and COPY FROM statements can arbitrarily create new columns as needed (what’s known as a dynamic column policy).